samples/react-native-app/README.md

```diff
@@ -0,0 +1,16 @@
+This project was bootstrapped with [Create React Native App](https://github.com/expo/create-react-native-app).
+
```

Comment:
Could you provide brain-dead simple instructions on how to get going? Include how to install deps in the project directory, etc.

Reply:
yep, done